### PR TITLE
fix static page and asset list header to top of viewport.

### PR DESCRIPTION
**Pull request by rjw57**
The asset list and static page headers scroll away as one scrolls the page. Re-style them to remain at the top of the page.

### CODE REVIEW

File: src/components/AssetListHeader.js [hunk 1]
```diff
@@ -12,19 +12,22 @@ const styles = theme => ({
...
   */
 const AssetListHeader = ({ title, classes }) => (
-  <AppBar position="static" className={classes.appBarBackground}>
+  <AppBar position="fixed" className={classes.appBar}>
     <Toolbar>
       <Typography variant="title" color="inherit" className={classes.flex}>
         { title }
       </Typography>
-      <Link className='App-raised-button-link' to="/asset/create">
+      <Link to="/asset/create">
```
Comment (joeadams04) [inline src/components/AssetListHeader.js]: Removing this class has introduced a small styling bug. As the button is wrapped in a link (<a />) it attempts to add text decoration. The removed class above looks to remove that (and incorrectly makes the text color black btw). Can we correctly set text-dec to none, and the text color to `coreGreen` like the same primary button on the form.

This was one of the small tasks on my list, but could probably fix in this PR.

File: src/containers/AssetList.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import AssetTable from '../components/AssetTable';
 import Page from '../containers/Page';
 import GetMoreAssets from '../components/GetMoreAssets';
 import { withRouter } from 'react-router'
+import { withStyles } from 'material-ui/styles';
 import { connect } from 'react-redux';
 import { getAssets, Direction } from '../redux/actions/assetRegisterApi';
```
Comment (joeadams04) [inline src/containers/AssetList.js]: We can remove the css import here if removing the last class.

Comment (joeadams04) [reply src/containers/AssetList.js]: Not shown in the preview above, but I'm referring to: `import '../style/App.css';`

File: src/components/AssetListHeader.js (hunk 1 above)
Comment (rjw57) [reply src/components/AssetListHeader.js]: Sure. Will fix.

File: src/containers/AssetList.js (hunk 2 above)
Comment (rjw57) [reply src/containers/AssetList.js]: wilco

**Comment by rjw57** [comment]
Since this had passed Joe's verbal review, I hit the green button.

**Pull request merged**